bot.go

```diff
@@ -42,7 +42,7 @@ func New(h *Handlers) *Bot {
 
 func (b *Bot) startPeriodicCommands() {
 	for _, config := range periodicCommands {
-		go func(config PeriodicConfig) {
+		func(b *Bot, config PeriodicConfig) {
```

Comment:
It's still necessary otherwise it will always use the `config` object of the last registered periodic command when running any of the crons.